etc/JDownloader.profile

```diff
@@ -32,14 +41,9 @@ seccomp
 shell none
 
 disable-mnt
```

Comment:
should other drives be disabled? if this is a generic downloader people might want download on another drive